application/datamanager/Dockerfile

```
@@ -1,19 +1,18 @@
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends libgomp1 \
+    apt-get install -y --no-install-recommends libgomp1 curl \
     && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /app
 
 COPY pyproject.toml ./
 
-COPY application/datamanager/ ./application/datamanager/
+COPY ./src / ./app
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)